panoptes_aggregation/routes.py

```diff
@@ -115,6 +117,24 @@ def index():
     for route, route_function in panoptes.panoptes.items():
         application.route('/panoptes/{0}'.format(route), methods=['POST', 'PUT'])(lambda: route_function(request.args.to_dict(), request.get_json()))
 
+    @application.route('/run_aggregation', methods=['POST'])
+    def run_aggregation():
+        content = request.json
+        project_id = content['project_id']
+        workflow_id = content['workflow_id']
+        user_id = content['user_id']
+        task = batch_aggregation.run_aggregation.delay(project_id, workflow_id, user_id)
```

Comment:
Does the code check anywhere that the `user_id` is a collaborator/owner on the project before allowing anything to run?

Reply:
Good point. The Panoptes request that initiates the external API call is protected, but this should be too. I added a check against the project's collaborators and cancel the task with `sys.exit()` if it fails.

Reply:
Putting this check in the worker since it seems overly heavy for the route.